babel/messages/frontend.py

```diff
@@ -852,6 +853,247 @@ def run(self):
+            raise OptionError('you must specify the output file')
+
+        if self.no_wrap and self.width:
+            raise OptionError("'--no-wrap' and '--width' are mutually exclusive")
+        if not self.no_wrap and not self.width:
+            self.width = 76
+        elif self.width is not None:
+            self.width = int(self.width)
+
+        if self.more_than is None:
+            self.more_than = 0
+        else:
+            self.more_than = int(self.more_than)
+        if self.less_than is not None:
+            self.less_than = int(self.less_than)
+        if self.unique:
+            self.less_than = 2
+
+    def _prepare(self):
+        templates: list[tuple[str, Catalog]] = []
+        message_info = {}
+
+        for filename in self.input_files:
+            with open(filename, 'r') as pofile:
+                template = read_po(pofile)
+            for message in template:
+                if message.id not in message_info:
+                    message_info[message.id] = {
+                        'count': 0,
+                        'strings': set(),
+                    }
+                message_info[message.id]['count'] += 1
+                message_info[message.id]['strings'].add(message.string if isinstance(message.string, str) else tuple(message.string))
+            templates.append((filename, template, ))
+
+        return templates, message_info
+
+    def run(self):
+        catalog = Catalog(fuzzy=False)
+        templates, message_info = self._prepare()
+
+        for path, template in templates:
+            if catalog.locale is None:
+                catalog.locale = template.locale
+
+            for message in template:
+                if not message.id:
+                    continue
+
+                count = message_info[message.id]['count']
+                diff_string_count = len(message_info[message.id]['strings'])
+                if count <= self.more_than or (self.less_than is not None and count >= self.less_than):
+                    continue
+
+                if count > 1 and not self.use_first and diff_string_count > 1:
+                    file_name = os.path.basename(path)
+                    catalog.add_conflict(message, file_name, template.project, template.version)
+
+                catalog[message.id] = message
+
+        catalog.fuzzy = any(message.fuzzy for message in catalog)
+
+        with open(self.output_file, 'wb') as outfile:
+            write_po(
+                outfile,
+                catalog,
+                width=self.width,
+                sort_by_file=self.sort_by_file,
+                sort_output=self.sort_output,
+                no_location=self.no_location,
+            )
+
+
+class MergeCatalog(CommandMixin):
+    description='updates translation PO file by merging them with updated template POT file with using compendium'
+    user_options=[
+        ('input-files', None, 'def.po (obsolete translations) ref.pot (actual template)'),
```

Comment:
```suggestion
        ('input-files', None, 'exactly two input files: def.po (obsolete translations); ref.pot (current template)'),
```

I assume "actual" means "current" here (common [false friend](https://en.wikipedia.org/wiki/False_friend)) :)